managed/cmd/pmm-managed/main.go

```diff
@@ -108,14 +109,17 @@ import (
 	"github.com/percona/pmm/version"
 )
 
+var (
+	interfaceToBind = env.GetEnv("PMM_DEV_BIND_INTERFACE", "127.0.0.1")
```

Comment:
Let's use `PERCONA_TEST_` prefix, it's standard naming in PMM for env variables that should not be used by user.

Reply:
Also probably it makes sense to validate value

Reply:
sure